### ISSUE TITLE
Error in Datamaps.prototype.updateChoropleth

### ISSUE THREAD

**Issue by rolmovel** [markmarkoh/datamaps#234]
Trying to integrate Datamaps with Zeppelin, using the angular-datamps plugin in order to use Datamaps under the angular framework, when trying to color countries using the fillColor key instead of fillKey key finally goes black instead.
After debugging, seems to be a bug in the updateChoropleth method. If you change the original:
```
color = this.options.fills[ subunitData.fillKey ]; (line 12155 of the master version)
```
for something like this:
```
color = val (subunitData.fillKey, subunitData.fillColor);
```
it seems to be solved.
Is there any collateral effect?

Thanks

**Comment by jensluch** [comment]
Same story(

**Comment by jensluch** [comment]
Found 1 issue. To fill any color you want you can change 'fills' object in this way:
map.updateChoropleth({ UKR:{fillColor: "red"}}) 
to
map.updateChoropleth({ UKR: "red"})

**Comment by markmarkoh** [comment]
Closed with #240
